### PR TITLE
Add max spill level in RowNumber fuzzer

### PR DESCRIPTION
**Pull request by duanmeng**
Add a max spill level config in the RowNumberFuzzer.

### CODE REVIEW

File: velox/exec/fuzzer/RowNumberFuzzer.cpp [hunk 1]
```diff
@@ -281,6 +283,7 @@ RowVectorPtr RowNumberFuzzer::execute(
   if (injectSpill) {
     spillDirectory = exec::test::TempDirectoryPath::create();
     builder.config(core::QueryConfig::kSpillEnabled, true)
+        .config(core::QueryConfig::kMaxSpillLevel, FLAGS_max_spill_level)
```
Comment (xiaoxmeng) [inline velox/exec/fuzzer/RowNumberFuzzer.cpp:286]: Can you randomize this in test instead of using a flag?

**Review by xiaoxmeng — COMMENTED** [review]
@duanmeng thanks!

File: velox/exec/fuzzer/RowNumberFuzzer.cpp (hunk 1 above)
Comment (duanmeng) [reply velox/exec/fuzzer/RowNumberFuzzer.cpp]: Got it, fixed.

**Review by xiaoxmeng — APPROVED** [review]
@duanmeng thanks!

**Pull request closed without merging**